app/Http/Requests/StorePrimaryWebsiteRequest.php

```diff
@@ -60,9 +68,25 @@ public function rules(): array
     public function withValidator(Validator $validator): void
     {
         $validator->after(function (Validator $validator) {
+            if (filled($this->input('url')) && ('custom' === $this->input('website_type'))) {
+                if (!filter_var($this->input('url'), FILTER_VALIDATE_URL)) {
+                    $validator->errors()->add('url', __("L'indirizzo inserito non è corretto."));
+                }
+                $host = parse_url($this->input('url'), PHP_URL_HOST);
+                if ($host && !$this->checkIsNotPrimary($host)) {
+                    $validator->errors()->add('url', __("L'indirizzo inserito appartiene a un'altra pubblica amministrazione."));
+                }
+
+                $publicAdministration = $this->input();
+                $publicAdministration['name'] = $publicAdministration['public_administration_name'];
+                $this->publicAdministration = $publicAdministration;
+                $this->isCustomPublicAdministration = true;
+            }
+
```

Comment:
Qui non c'è nessun controllo preventivo sul fatto che la funzionalità per le PA custom sia attivata in configurazione.